code/datums/mind/_mind.dm

```diff
@@ -200,8 +200,7 @@
 	QDEL_NULL(antag_hud)
 	new_character.mind = src //and associate our new body with ourself
 	antag_hud = new_character.add_alt_appearance(/datum/atom_hud/alternate_appearance/basic/antagonist_hud, "combo_hud", src)
-	for(var/a in antag_datums) //Makes sure all antag datums effects are applied in the new body
-		var/datum/antagonist/A = a
+	for(var/datum/antagonist/A as anything in antag_datums) //Makes sure all antag datums effects are applied in the new body
 		A.on_body_transfer(old_current, current)
```

Comment:
are you a bad enough dude to fix this single-letter var while you're here? no obligation but i figure it's simple enough

Reply:
yea sure